supabase/migrations/20231011172312_rosetta_init.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TYPE mapper_status AS ENUM (
+    'PENDING_CREATION',
+    'PENDING_UPDATE',
+    'IN_PROGRESS',
+    'PENDING_APPROVAL',
+    'ACTIVE',
+    'INACTIVE',
+    'DECLINED',
+    'FAILED'
+);
+CREATE TABLE rosetta_mappers (
+    id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    key TEXT NOT NULL,
+    -- A composite key (e.g., "OpenAI:GPT3.5")
```

Comment:
Is this still true? Or are we moving to path?

Reply:
Not true anymore, comment isn’t necessary